core/src/main/java/software/amazon/awssdk/core/async/FileAsyncResponseHandler.java

```diff
@@ -82,6 +82,7 @@ public ResponseT complete() {
     /**
      * {@link Subscriber} implementation that writes chunks to a file.
      */
+    // FIXME cover with Reactive Streams TCK, looks ok from a first brief look, but could be missing some edge cases
```

Comment:
This PR is not (at least yet), intended to be merged, please assume those are just markers for myself to know where/how many implementations are there and what we should cover with tests.